frontend/assessments/2/public/index.scss

```diff
@@ -0,0 +1,414 @@
+    margin: 10px 5px;
+    padding: 15px;
+    font-size: x-large;
+    font-weight: 600;
+    color: white;
+}
+
+#main{
+    display: flex;
+    justify-content: center;
+    width: 100%;
+}
+
+.info{
+    flex: 3;
+    display: flex;
+    flex-direction: column;
+}
+
+.history{
+    flex: 1;
+    border: 1px solid #000;
+    border-radius: 4px;
+    padding: 10px;
+    margin-top: 15px;
+    margin-left: 15px;
+    margin-right: 15px ;
+    height: 635px;
+    display: flex;
+    flex-direction: column;
+}
+
+.stock-info{
+    margin-top: 15px;
+    margin-left: 15px;
+    height: 125px;
+    margin-bottom: 10px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.grid{
+    border: 1px solid #000;
+    margin-left: 15px;
+    height: 500px;
+}
+
+.stock-name{
+    flex: 2;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+} 
+
+.stock-price{
+    flex: 3;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.stock-qty{
+    flex: 1.5;
+}
+
+.buy, .sell{
+    flex: 1;
+}
+
+.stock-name, .stock-price, .stock-qty, .buy, .sell{
+    text-align: center;
+}
+
+#buy-btn{
+    color: #2f9e44;
+    border: 1px solid #2f9e44;
+    background-color: #b2f2bb;
+    padding: 20px;
+    cursor: pointer;
+}
+
+#sell-btn{
+    color: #e03131;
+    border: 1px solid #e03131;
+    background-color: #ffc9c9;
+    padding: 20px;
+    cursor: pointer;
+}
+
+#zomato_logo{
+    height: 40px; 
+    width: 50px;
+    margin-right: 10px;
+}
+
+#zomato-tag{
+    font-size: x-large;
+}
+
+.stock-price > p{
+    margin: 0 4px;
+    font-size: x-large;
+}
+
+#upOrDown{
+    font-size: xx-large;
+}
+
+#percentChange{
+    font-size: medium;
+}
+
+#qty-input{
+    width: 72%;
+    padding: 15px;
+    font-size: large;
+}
+
+#history-tag{
+    margin: 5px;
+    margin-bottom: 10px;
+    font-size: larger;
+}
+
+.history-div{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin: 8px 0;
+    padding: 8px;
+    border: 1px solid #000;
+    border-radius: 10px;
+}
+
+.history-price-qty-date{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+}
+
+.history-price-qty{
+    font-weight: 600;
+}
+
+.history-buy-txn{
+    text-align: end;
+    color: #2f9e44;
+    flex: 1;
+}
+
+.history-sell-txn{
+    text-align: end;
+    color: #e03131;
+    flex: 1;
+}
+
+#up{
+    color: #2f9e44;
+}
+
+#down{
+    color: #e03131;
+}
+
+.grid{
+    display: flex;
+    align-items: end;
+}
+
+.grid > div{
+    display: flex;
+    align-items: end;
+    width: 100px;
+    border-right: 1px dotted gray;
+    height: 500px;
+}
+
+.grid-ele{
+    width: 20px;
+}
+
+.green-bar{
+    border: 1px solid #2f9e44;
+    background-color: #b2f2bb;
+}
+
+.red-bar{
+    border: 1px solid #e03131;
+    background-color: #ffc9c9;
+}
+
+
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
```

Comment:
Follow a proper architecture.
Imports at top.

Use tag hierarchy and nest them accordingly for clean code.